client/src/components/AuditAdmin/Progress/JurisdictionDiscrepancies.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import { Classes, Colors, Dialog, H6, HTMLTable } from '@blueprintjs/core'
+import styled from 'styled-components'
+import {
+  IJurisdiction,
+  useDiscrepanciesByJurisdiction,
+} from '../../useJurisdictions'
+import useContestsJurisdictionAdmin from '../../JurisdictionAdmin/useContestsJurisdictionAdmin'
+import { IContest } from '../../../types'
+
+const ContestDiscrepanciesTable = styled(HTMLTable).attrs({
+  bordered: true,
+  striped: true,
+})`
+  &.${Classes.HTML_TABLE} {
+    background: #ffffff;
+    border: 1px solid ${Colors.LIGHT_GRAY1};
+    margin-bottom: 32px;
+    table-layout: fixed;
+    width: 100%;
+  }
+
+  &.${Classes.HTML_TABLE} td {
+    vertical-align: middle;
+  }
```

Comment:
Did you look at the various styled table options in components/Atoms/Table.tsx? There might be one that works well here, but if not I'm fine with keeping what you have

Reply:
StyledTable
<img width="1512" alt="Screenshot 2024-11-04 at 3 35 32 PM" src="https://github.com/user-attachments/assets/cfd9d54a-cfff-43d9-8b10-b92b1a73da1f">

FlexTable
<img width="1424" alt="Screenshot 2024-11-04 at 3 29 53 PM" src="https://github.com/user-attachments/assets/de35f955-3ae9-4dc0-87f8-5b51d7256c1e">

HTMLTable
<img width="1512" alt="Screenshot 2024-11-04 at 3 34 08 PM" src="https://github.com/user-attachments/assets/2eb648a4-9e6f-4135-8781-b80b10dfa982">

I hadn't, so just mocked the three up. I don't have a strong leaning (though the new options could definitely be improved with some borders etc.). I'm probably in between the current and the StyledTable (with border added) due to the row striping.

Reply:
Thanks for investigating. I'm now realizing we haven't been that consistent in using these, and I actually like the current design the best haha. Sorry for the detour!

Reply:
No worries, was good to see those options! Will continue as is